tests/foreman/ui/test_ansible.py

```diff
@@ -471,15 +468,14 @@ def test_positive_non_admin_user_access_with_usergroup(
         user = target_sat.api.User(
             login=gen_string('alpha'), password=password, admin=False
         ).create()
-        request.addfinalizer(user.delete)
```

Comment:
why removing user.delete from finalizer, this looks unrelated?

Reply:
The issue is causing conflicts and leading to test failures. After removing this, the tests are running successfully

Reply:
what conflict do you see? its different entity, not sure what conflict it may cause

Reply:
![Screenshot from 2024-09-11 00-48-40](https://github.com/user-attachments/assets/b6a764f2-8fe1-4780-adf1-9c230b81c471)

Reply:
Getting this error - `failed on teardown with "requests.exceptions.HTTPError: 422 Client Error: Unprocessable Content for url: https://ip-10-0-216-32.rhos-01.prod.psi.rdu2.redhat.com/api/v2/users/5"`
Any other solution for this?

Reply:
@Gauravtalreja1 What is the reason to not delete the user in teardown?

Reply:
I think this question is for @amolpati30

Reply:
@amolpati30 Deleting the user in teardown should work.

Reply:
@amolpati30 so could you add the finalizer back?

Reply:
I have re-added the finalizer, made some adjustments, and it is now working.